src/message-stream.ts

```diff
@@ -151,6 +152,23 @@ export class MessageStream extends PassThrough {
     );
     this._keepAliveHandle.unref();
   }
+
+  /**
+   * Updates the stream ack deadline with the server.
+   *
+   * @param {Duration} deadline The new deadline value to set.
+   */
+  setAckDeadline(deadline: Duration) {
```

Comment:
setStreamAckDeadline?

Reply:
I figured since it's in MessageStream, it's probably understood, but there's no harm in being explicit about it :)